tokio/src/task/blocking.rs

```diff
@@ -109,18 +109,5 @@ where
     F: FnOnce() -> R + Send + 'static,
     R: Send + 'static,
 {
-    #[cfg(feature = "tracing")]
-    let f = {
-        let span = tracing::trace_span!(
-            target: "tokio::task",
-            "task",
-            kind = %"blocking",
-            function = %std::any::type_name::<F>(),
-        );
-        move || {
-            let _g = span.enter();
-            f()
-        }
-    };
     crate::runtime::spawn_blocking(f)
```

Comment:
This code compiled before we even added `Runtime::spawn_blocking` to the codebase, so it cannot be true that it just calls that function.

Reply:
`crate::runtime::spawn_blocking` is a reexport for `crate::runtime::blocking::pool::spawn_blocking` which is basically `crate::task::spawn_blocking`, yes. It's not `crate::runtime::Runtime::spawn_blocking`

Reply:
`crate::runtime::Runtime::spawn_blocking` is basically a copy of `crate::runtime::blocking::pool::spawn_blocking`, except the former is already inside a Runtime, so it uses its internal handle, whilst the latter gets the current handle and uses it.
We could avoid the duplication by implementing this on Handle directly.

Moving the tracing integration around isn't the right call here though. We should either copy it or move that into Handle

Reply:
Looks like I got confused by all the instances of `spawn_blocking`. I'll dig into this again tonight with the suggestions. Thank you for the review.